modules/ivc_champva/app/form_mappings/vha_10_10d.json.erb

```diff
@@ -11,60 +11,60 @@
-  "form1[0].#subform[0].MI3[0]":                         "<%= form.data['applicants'][2]&.dig('full_name', 'middle') %>",
+  "form1[0].#subform[0].MI3[0]":                         "<%= form.data['applicants'][2]&.dig('full_name', 'middle')&.first %>",
   "form1[0].#subform[0].SSN3[0]":                        "<%= form.data['applicants'][2]&.dig('ssn_or_tin') %>",
   "form1[0].#subform[0].StreetAddress3[0]":              "<%= form.data['applicants'][2]&.dig('address', 'street') %>",
   "form1[0].#subform[0].City3[0]":                       "<%= form.data['applicants'][2]&.dig('address', 'city') %>",
   "form1[0].#subform[0].ZipCode3[0]":                    "<%= form.data['applicants'][2]&.dig('address', 'postal_code') %>",
   "form1[0].#subform[0].DateOfBirth3[0]":                "<%= form.data['applicants'][2]&.dig('date_of_birth')  %>",
   "form1[0].#subform[0].EmailAddress3[0]":               "<%= form.data['applicants'][2]&.dig('email') %>",
   "form1[0].#subform[0].PhoneNumber3[0]":                "<%= form.data['applicants'][2]&.dig('phone_number') %>",
-  "form1[0].#subform[0].Gender3[0]":                     "<%= form.data['applicants'][2]&.dig('gender') ? 1 : 0 %>",
+  "form1[0].#subform[0].Gender3[0]":                     "<%= form.data['applicants'][2]&.dig('gender') == 'male' ? 0 : 1 %>",
   "form1[0].#subform[0].EnrolledMedicare[2]":            "<%= form.data['applicants'][2]&.dig('is_enrolled_in_medicare') ? 1 : 0  %>",
   "form1[0].#subform[0].HasOtherInsurance[2]":           "<%= form.data['applicants'][2]&.dig('has_other_health_insurance') ? 1 : 0  %>",
   "form1[0].#subform[0].RelationshipToVeteran3[0]":      "<%= form.data['applicants'][2]&.dig('vet_relationship') %>",
   "form1[0].#subform[0].State3[0]":                      "<%= form.data['applicants'][2]&.dig('address', 'state') %>",
   "form1[0].#subform[0].DateSigned[0]":                  "<%= form.data.dig('certification', 'date') %>",
   "form1[0].#subform[0].SignatureField11[0]":            "<%= form.data['statement_of_truth_signature'] %>",
-  "form1[0].#subform[0].LastName4[0]":                   "<%= form.data.dig('certification', 'lastName') %>",
-  "form1[0].#subform[0].FirstName4[0]":                  "<%= form.data.dig('certification', 'firstName') %>",
-  "form1[0].#subform[0].MI4[0]":                         "<%= form.data.dig('certification', 'middleInitial') %>",
-  "form1[0].#subform[0].StreetAddress4[0]":              "<%= form.data.dig('certification', 'streetAddress') %>",
+  "form1[0].#subform[0].LastName4[0]":                   "<%= form.data.dig('certification', 'last_name') %>",
+  "form1[0].#subform[0].FirstName4[0]":                  "<%= form.data.dig('certification', 'first_name') %>",
+  "form1[0].#subform[0].MI4[0]":                         "<%= form.data.dig('certification', 'middle_initial')&.first %>",
+  "form1[0].#subform[0].StreetAddress4[0]":              "<%= form.data.dig('certification', 'street_address') %>",
```

Comment:
Updated to use snake case, which was preventing these values from printing previously